lib/Dialect/Triton/IR/Types.cpp

```diff
@@ -157,7 +157,15 @@ Type getPointerTypeSameShape(Type type) {
   }
 }
 
-Type getPointerType(Type type) { return PointerType::get(type, 1); }
+Type getPointerType(Type type, int addressSpace) {
```

Comment:
Why not just using `PointerType::get(type, addressSpace);`?

Reply:
below is just what you posted.
keep the existing function `getPointerType` here to make minimal change.

Reply:
OK, I think the interface will probably be confusing for developers by providing an option to change the address space.

I would suggest adding a comment to the interface at least, saying something like, "The upstream Triton only uses address space 1." Otherwise, developers may think that Triton supports address space 3.

Reply:
done